cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -489,3 +490,385 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            3. Test that snapshot can be created from Snapshot page
+                * Navigate to My Services -> <service_name> -> <vm_name> -> Snapshots
+                * Click Configuration -> Create Snapshot
+                * Fill Name and click Create
+        expectedResults:
+            1. Snapshot successfully created
+            2. Snapshot successfully created
+            3. Snapshot successfully created
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([InfraProvider])
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_snapshot_timeline_group_actions():
+    """
+    Test the SUI snapshot timeline.
+    Test grouping of actions in a timeline. Try to create a couple of
+    snapshots in a rapid succession, check how it looks in the timeline.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/3h
+        setup:
+            1. Add infra provider
+        testSteps:
+            1. create a new vm
+            2. create multiple snapshots in fast succession (two should be enough)
+            3. go to the VM details page, then Monitoring -> Timelines
+            4. select "Management Events" and "Snapshot Activity" and click Apply
+            5. click on the group of events in timeline
+        expectedResults:
+            1. vm created
+            2. snapshots created
+            3. timelines page displayed
+            4. group of events displayed in the timeline
+            5. details of events displayed, correct number of events
+               displayed, time/date seems correct
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([InfraProvider])
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_snapshot_timeline_new_vm():
+    """
+    Test the SUI snapshot timeline.
+    See if there"s no timeline when there"s no snapshot.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/6h
+        setup:
+            1. Add infra provider
+        testSteps:
+            1. create a new vm
+            2. go to the VM details page, then Monitoring -> Timelines
+            3. select "Management Events" and "Snapshot Activity" and click Apply
+        expectedResults:
+            1. vm created
+            2. timelines page displayed
+            3. no timeline visible, warning "No records found for this timeline" displayed
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([InfraProvider])
+@test_requirements.snapshot
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_ssui_snapshot_memory_checkbox():
+    """
+    Test "snapshot vm memory" checkbox when creating snapshot for powered off vm.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Provisioning
```

Comment:
Change case component to SUI if the test is for Self Service.

Reply:
casecomponents should be fixed now.